src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/DefaultApiAlerts.apitargets

```diff
@@ -0,0 +1,33 @@
+﻿# Target syntax messages are defined as an ID and message (formatted as `ID: Message`) followed by APIs that the message applies to.
+# APIs are described by a three-tuple of API type, API name, and a boolean indicating whether the alert should be given in cases where
+# the API is only ambiguously matched.
+
+A: HTTP modules and HTTP handlers should be rewritten as middleware in ASP.NET Core. https://docs.microsoft.com/aspnet/core/migration/http-modulesIHttpModule
+Type, System.Web.IHttpModule, true
+Type, System.Web.IHttpHandler, true
+
+B: WCF server APIs are unsupported on .NET Core. Consider rewriting to use gRPC (https://docs.microsoft.com/dotnet/architecture/grpc-for-wcf-developers), ASP.NET Core, or CoreWCF (https://github.com/CoreWCF/CoreWCF) instead.
+Type, System.ServiceModel.ServiceHost, true
```

Comment:
comma delimited could break with generic types, right?

Reply:
Currently, when we get the name of the type in the analyzer, we're leaving off any generic parameters. It would probably be more correct to get the name of the open/unbound generic type to reduce ambiguity, but currently we just get the generic type's identifier so there won't be any commas. 

I can change this to semi-colon delimiting, though, so that it doesn't break if we change things in the future. Also, I'll add a comment about how generics are handled.

Reply:
Actually, thinking about it more I'd like to just handle generics properly. I'll update the way we get the name for the generic to use "ildasm" style generics List`1, Dictionary`2, etc. since we don't know what generic parameter names make sense. Then commas will still be good delimiters. I'll include a sample and comment in the apitargets file.